packages/core/src/pubsub/incoming-channel.ts

```diff
@@ -24,7 +24,7 @@ function buildResubscribeQueue(logger: DiagnosticsLogger) {
       // For some reason the first call to pubsub.subscribe throws this error.
       retry();
     } else {
-      logger.err(error);
+      logger.err(error.toString());
```

Comment:
This doesn't seem right.. `logger.err` should take in an error and handle it correctly.

See: https://github.com/seanpmaxwell/overnight/blob/master/src/logger/lib/Logger.ts#L213-L215